hazelcast/src/main/java/com/hazelcast/map/impl/operation/steps/MultipleEntryOpSteps.java

```diff
@@ -56,7 +55,7 @@ public void runStep(State state) {
             }
 
             if (!keysToLoad.isEmpty()) {
-                state.setKeysToLoad(Collections.unmodifiableList(keysToLoad));
+                state.setKeysToLoad(keysToLoad);
```

Comment:
Why this change is made?

Reply:
Failed with new tests that i added in this PR. And also i can see no reason to make it unmodifiable.